### DISCUSSION TITLE
when exporting app, BB can not import the file it just exported

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1515] [Help]
Hi I am having trouble. When I export an app I have built the file is no longer able to be read by the BB client. When I try and import the app from the file I just exported. BB dose not recognize the app it just exported. Is this a bug or a settings problem?

Thank you

**Comment by mjashanks** [comment]
Hey Andrew. Are you able to share an error message with us? 
- Open dev tools : `Ctrl + Shift +I` for windows or `Cmd + Opt + I` on mac
- Click on the `console` tab, in dev tools
- Import your app.

If you could screenshot that, it would be really helpful.

Also, I'd be happy to try the import myself to debug it, if you are able to share your export file via email or our discord chat.

**Reply by KernOil** [reply]
here is the error I am getting with the newly exported app trying to re import it. 

![image](https://user-images.githubusercontent.com/80058238/119151910-ac670400-ba04-11eb-8ac7-afdf9376ff87.png)

**Reply by KernOil** [reply]
I emailed you a copy of my exported app.
